airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -112,6 +112,61 @@ public JsonNode toJdbcConfig(final JsonNode config) {
     return Jsons.jsonNode(configBuilder.build());
   }
 
+  private List<String> getAdditionalParameters(final JsonNode config) {
+    final List<String> additionalParameters = new ArrayList<>();
+    addJDBCUrlParameters(config, additionalParameters);
+    if (useSSL(config)) {
+      addSSLJDBCParameters(additionalParameters);
+    }
+    return additionalParameters;
+  }
+
+  private void addJDBCUrlParameters(final JsonNode config, final List<String> additionalParameters) {
+    if (config.has(JDBC_URL_PARAMS_KEY)) {
+      final String additionalParams = config.get(JDBC_URL_PARAMS_KEY).asText();
+      if (additionalParams.isBlank()) {
+        return;
+      }
+      if (useSSL(config)) {
+        for (final String p : getSSLParameters().keySet()) {
+          if (containsParameterKey(additionalParams, p)) {
+            throw new RuntimeException("Cannot overwrite JDBC parameter " + p);
+          }
+        }
+      }
+      additionalParameters.add(additionalParams);
+    }
+  }
+
+  private boolean containsParameterKey(final String additionalParametersString, final String key) {
+    final String s = String.format("%s=", key);
+    return additionalParametersString.contains(s);
+  }
+
+  private void addSSLJDBCParameters(final List<String> additionalParameters) {
+    for (final Entry<String, String> sslParameter : getSSLParameters().entrySet()) {
+      final String param = formatParameter(sslParameter.getKey(), sslParameter.getValue());
+      additionalParameters.add(param);
+    }
+  }
+
+  private boolean useSSL(final JsonNode config) {
+    return !config.has("ssl") || config.get("ssl").asBoolean();
+  }
+
+  static Map<String, String> getSSLParameters() {
+    final Builder<String, String> builder = ImmutableMap.builder();
+    return builder
+        .put("useSSL", "true")
+        .put("requireSSL", "true")
+        .put("verifyServerCertificate", "false")
+        .build();
+  }
```

Comment:
Since this is a constant, it can be put in a static variable.

Reply:
moved to a constant.